services/drupal/web/modules/custom/epa_workflow/epa_workflow.module

```diff
@@ -647,6 +647,39 @@ function epa_workflow_module_implements_alter(&$implementations, $hook) {
  */
 function epa_workflow_form_revision_overview_form_alter(&$form, FormStateInterface $form_state, $form_id) {
   $form['node_revisions_table'][0]['operations']['#prefix'] = '<strong>Latest revision</strong><br>';
+
+  $node = $form_state->getBuildInfo()['args'][0];
+  if ($node instanceof NodeInterface && !$node->isNew()) {
+    // Loading will always return the "Current revision".
+    /** @var \Drupal\node\NodeInterface $current */
+    $current = \Drupal::entityTypeManager()
+      ->getStorage('node')
+      ->load($node->id());
+
+    // If editing latest revision and has a Publish Date set.
+    if ($node->getRevisionId() == $current->getRevisionId() && isset($node->field_publish_date->value)) {
```

Comment:
I assume (though I'm not totally sure) that the $node object here is the latest revision.  If so, why are we checking to see if it matches the current revision?  We want all of the logic here to apply regardless of whether the current and latest revision are the same, unless I'm missing something.